packages/node_modules/@ciscospark/widget-recents/src/container.js

```diff
@@ -648,22 +341,7 @@ export class RecentsWidget extends Component {
       } = widgetError);
     }
 
-    const isWebRTCSupported = media.getIn(['webRTC', 'isSupported']);
-
-    if (spacesListArray &&
-      hasFetchedSpaces &&
-      hasFetchedGroupMessageNotificationFeature &&
-      hasFetchedMentionNotificationFeature) {
-      const spacesListWithActivityText = spacesListArray.map((space) => {
-        const s = Object.assign({}, space);
-
-        s.activityText = this.constructActivityText(space);
-
-        return s;
-      });
-
-      const handleCallClick = isWebRTCSupported ? this.handleSpaceCallClick : undefined;
```

Comment:
We shouldn't care if webrtc is supported, if the user clicks the meeting in progress timer, we send an event saying so and that's all.